config.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package release;
+
+service Release {
+    // Get information about a release from its id
+    rpc GetRelease(string) returns (Release) {};
+    // List all releases matching the request
+    rpc ListReleases (ListReleasesRequest) returns (ListReleasesReponse) {};
+}
+
+message ListReleasesRequest {
+    // List only releases created after, if != 0
+    int64 created_after = 1;
+    // 
```

Comment:
Better add a filter object in the requests to capture the criteria

Reply:
Done